data/pathfinder/dreamscarred_press/path_of_war/pow_abilities_psi.lst

```diff
@@ -4,15 +4,15 @@
+# Ability Name				Unique Key					Category of Ability		Type											Define																																																																																										Description																																																																																																																																																																																																																																																																																						Ability																																																																																																																																																																																																													Bonus Ability Pool				Bonus DC				Modify VAR																																																																																																																																																																																																																																																																		Source Page
+Pathwalker Initiating			KEY:Pathwalker ~ Initiating		CATEGORY:Special Ability	TYPE:PsychicWarriorClassFeatures.ClassFeatures.SpecialQuality																																																																																												DESC:Initiator Level: %1, Maximum Maneuver Level:%2|PathwalkerInitiatorLVL|PathwalkerHighestManeuverLevel																																																																																																																																																																																																																																																																						ABILITY:Internal|AUTOMATIC|Pathwalker ~ Initiator Variables	ABILITY:Psychic Warrior Class Feature|AUTOMATIC|Pathwalker ~ Maneuvers|PREVARGTEQ:PsychicWarriorLVL,1	ABILITY:Psychic Warrior Class Feature|AUTOMATIC|Pathwalker ~ Maneuvers Readied|PREVARGTEQ:PsychicWarriorLVL,1	ABILITY:Psychic Warrior Class Feature|AUTOMATIC|Pathwalker ~ Stances Known|PREVARGTEQ:PsychicWarriorLVL,1	ABILITY:Internal|AUTOMATIC|Pathwalker ~ Disciplines|PREVARGTEQ:PsychicWarriorLVL,1																																																																																																																																								BONUS:DC|CLASS.Pathwalker|WIS																																																																																																																																																																																																																																																																				SOURCEPAGE:p.132
```

Comment:
I've moved spellcasters from `Pathwalker Initiating` ability to the first martial warrior paths so that spellcasters are added when it is guaranteed that at least one discipline was chosen.
To make maneuvers and stances appear in spell list, you have to update classes. For example, add and then remove any class. I don't know how to fix that.